SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/OptOutServices/EmailConfirmationDataService.swift

```diff
@@ -19,12 +19,28 @@
 import Foundation
 import Common
 import Algorithms
+import os.log
+
+public protocol EmailConfirmationDataChecking {
+    var emailConfirmationDataService: EmailConfirmationDataServiceProvider? { get }
+    func checkForEmailConfirmationData() async
+}
+
+extension EmailConfirmationDataChecking {
```

Comment:
I think I need selling on this, what's the purpose of squashing the error like this?